Modulefile

```diff
@@ -7,6 +7,6 @@ summary 'Minecraft / CraftBukkit Server v2'
 description 'Install and manage a Minecraft or CraftBukkit server'
 project_page 'https://github.com/andschwa/puppet-minecraft'
 
-dependency 'maestrodev/wget', '>= 1.1.0'
+dependency 'puppet/archive', '>= 0.5.1'
```

Comment:
* namespace: go with `voxpupuli`
* version: we will release archive 1.0.0 soon so we should already depend on it in metatadata.json (>= 1.0.0 < 2.0.0)
* can can bump this module to 2.0.0-rc0